understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
-      </section>
-      <section>
-      <h3>Viewing distance and display resolution</h3>
-        <p>The value of 320 CSS pixels was chosen as a reasonable minimum size that authors can achieve. This value lines up with the reported viewport width of small displays of common mobile devices. The width of 320 CSS pixels exactly corresponds to a desktop browser window set to a width of 1280px and zoomed in to 400%. It should be noted that 400% applies to the dimension, not the area. It means four times the default width and four times the default height.</p>
-      <figure id="figure-CSS-pixel-letter-4x"><img src="img/css-pixel-by-device.png" width="462" height="198" alt="Diagram showing the size of character needed by viewing distance to make the same image on the retina with small screen devices close, large screen devices further away."/>
-        <figcaption>A letter of the same CSS pixel size on different displays with different resolutions</figcaption>
-      </figure>
-        <p>When we read, the size of the print is not as important as the image it projects on the retina of our eye. Phones are designed for close viewing while desktops are designed for viewing farther away. As a consequence 16px print on a phone is physically smaller than 16px print on a desktop. This is not a problem because both print sizes cast the same image on our retina if they are viewed at their intended distance.</p>
-      </section>
-      <section>
-      <h3>Visibility and availability of content</h3> 
-      <p>How much of the content is visible may change at different scales. For example, navigation menus that are fully visible in the desktop layout are often collapsed into fewer items, or even into a single menu button (the 'hamburger' icon pattern) so they take up less screen space.</p>
-      <p>The Success Criterion is met as long as all content and functionality are still fully available -  either directly, or revealed via accessible controls, or accessible via direct links.</p>
-      </section>
+        <p>The intent of this success criterion is to let users enlarge (zoom in) text and other content without having to perform bi-directional scrolling to read. Lines of text that extend beyond the viewport in two-directions force users to scroll back-and-forth to read, which can cause them to lose their place, and significantly increase both physical and cognitive effort. Sections of content are expected to reflow within the appropriate sizing requirement defined by this success criterion, in regards to the text's intended direction of reading. With exceptions for sections of content which functionally requires bi-directional layout for usage or meaning.</p>
```

Comment:
"two-directions" is redundant. A line of text can only extend beyond the viewport in one direction. I've removed "zoom in" again because I find it disruptive to reading, and  the method of text enlargement is irrelevant to the need to reflow
```suggestion
        <p>The intent of this success criterion is to let users enlarge text and other content without having to perform bi-directional scrolling to read. Lines of text that extend beyond the edge of the viewport force users to scroll back and forth to read. This can cause them to lose their place, and significantly increases both physical and cognitive effort. Sections of content are expected to reflow within the appropriate sizing requirement defined by this success criterion. An exception exists for sections of content which functionally requires bi-directional layout for usage or meaning.</p>
```

Reply:
hmm. maybe worth talking about this a bit. i understand the suggestions you're making, but they do change part of the point i was trying to make.

Reply:
Mostly happy with the updates, but I think the second sentence is missing the bi-directional aspect. I.e. it's confusing because "beyond the viewport" could be down, but here you mean (and assume) side to side. (or vice versa)

> Text that extends beyond the edge of the viewport in two-directions forces users to scroll back and forth to read lines of text.

Reply:
made an update that incorporates most of these edits.  thank you both.  see my message in the commit.